Plugin/src/SofaPython3/PythonEnvironment.cpp

```diff
@@ -173,34 +173,34 @@ void PythonEnvironment::Init()
     if ( !Py_IsInitialized() )
     {
         msg_info("SofaPython3") << "Initializing python";
-        py::initialize_interpreter();
+        executePython([]{ py::initialize_interpreter(); });
         // the first gil aquisition should happen right after the python interpreter
         // is initialized.
         static const PyThreadState* init = PyEval_SaveThread(); (void) init;
     }
 
-    PyEval_InitThreads();
+    executePython([]{ PyEval_InitThreads(); });
```

Comment:
This is not python code (it is setting the python environment). I don't think it has to be in executePython.

Reply:
ok. By the way, this function is deprecated in Python 3.9 and will be disabled in 3.11.

Reply:
To be honnest I didn't followed the evolution of C-Python implementation so I have no idea what is the correct path to improve the implementation ins PythonEnvironement.